02_bubble/nicholeL/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+//Compare two adjacent numbers
+func bubbleSort(s []int) []int {
+	for index := 0; index < len(s); index++ {
+		for count := 0; count < len(s)-index-1; count++ {
+			if s[count] >= s[count+1] {
+				s[count], s[count+1] = s[count+1], s[count]
+			}
+		}
+	}
+	return s
+}
+
+func insertSort(s []int) []int {
+
```

Comment:
Unnecessary blank line.
No need for blank lines at beginning or end of block.
Use blank lines between type, method and function definitions and within a block for logical separation.

Reply:
thanks for your advice and will update